liblxqt/liblxqt_fr.ts

```diff
@@ -2,158 +2,187 @@
 <!DOCTYPE TS>
 <TS version="2.1" language="fr">
 <context>
+    <name>ConfigDialog</name>
+    <message>
+        <location filename="../configdialog/lxqtconfigdialog.ui" line="14"/>
+        <source>Dialog</source>
+        <translation>Dialogue</translation>
+    </message>
+</context>
+<context>
     <name>LXQt::MessageBox</name>
     <message>
-        <location filename="../../lxqtpowermanager.cpp" line="72"/>
```

Comment:
Of course, the change was not voluntary. It is lupdate which determines the paths. Which means I was not in the right directory when I used lupdate. But I would say that there are many pitfalls for translators.
I corrected for the 4 packages